scss/sections/_contact.scss

```diff
@@ -0,0 +1,69 @@
+#contact {
+	display: flex;
+	justify-content: center;
+	margin: 0 auto;
+	width: 100%;
+	max-width: 1400px;
+	.contact_container {
+		display: flex;
+		flex-direction: column;
+		width: 100%;
+		h1 {
```

Comment:
[REQ]: Make new line always where you declare selector.
This comment refers to all files.